rolling/ci-spaceros.yaml

```diff
@@ -0,0 +1,98 @@
+%YAML 1.1
+# ROS buildfarm ci-build file
+---
+build_environment_variables:
+  ROS_PYTHON_VERSION: '3'
+build_tool: colcon
+build_tool_args: '--cmake-args -DCMAKE_BUILD_TYPE=Debug -DEXPORT_COMPILE_COMMANDS=ON --no-warn-unused-cli'
+build_tool_test_args: '--retest-until-pass 2 --ctest-args -LE "(ikos|xfail)" --pytest-args -m "not xfail"'
```

Comment:
The ikos tests are only run as ctest tests but I could add a pytest mark for consistency if that is preferred.

Reply:
Seems like the consistency would be good. Does that mean that the IKOS tests would be run with either ctest or pytest? I'm not that familiar with the pytest marks.

Reply:
Items passed to --ctest-args will get passed through to any package that has tests run via ctest (all CMake-derived packages) and likewise --pytest-args gets passed to any python packages (I don't remember the exact facilities but as far as I know pytest is the default test "runner" for colcon-python-setup-py packages.)

In our workflow ament_ikos is only ever invoked as a ctest test. Invoking it via pytest would require a package author to write a custom pytest test using it, mark that test with the ikos label, and then it's unclear what that would even do since ROS packages at least don't readily support Python packages with native extensions (CMake or ament_cmake is used in that case). So the only reason to add the label exclusion is to prevent people from asking "why is this _here_ but not _there_".